packages/forms-web-app/src/dynamic-forms/has-questionnaire/questions.js

```diff
@@ -79,7 +81,7 @@ exports.questions = {
 		title: 'Who was notified',
 		question: 'Who did you notify about this application?',
 		fieldName: 'notified-who',
-		validators: []
+		validators: [new RequiredFileUploadValidator(), new MultifileUploadValidator()]
```

Comment:
this is just to show implementation of the two validators - can remove if required